modules/version.nix

```
@@ -0,0 +1,79 @@
+{ lib, pkgs, options, config, ... }:
+with builtins; with lib;
```

```suggestion
with lib;
```
almost anything under builtins is also available under lib